master/reference/calicoctl/resources/policy.md

```diff
@@ -33,6 +33,9 @@ spec:
       - 6379
   egress:
   - action: allow
+  types:
+  - ingress
+  - egress
```

Comment:
I would put `types` before `ingress` above.  That would read much more naturally.

Reply:
done.